distribution/pom.xml

```diff
@@ -54,7 +54,7 @@
 							<goal>wget</goal>
 						</goals>
 						<configuration>
-							<url>http://download.nextag.com/apache/felix/org.apache.felix.main.distribution-6.0.1.zip</url>
+							<url>http://mirrors.ibiblio.org/apache/felix/org.apache.felix.main.distribution-6.0.1.zip</url>
```

Comment:
I would not consider ibiblio.org very reliable. Why not using Maven Central (http://repo1.maven.org/maven2/org/apache/felix/org.apache.felix.main.distribution/6.0.1/org.apache.felix.main.distribution-6.0.1.zip) or Nexus (https://repository.jboss.org/org/apache/felix/org.apache.felix.main.distribution/6.0.1/org.apache.felix.main.distribution-6.0.1.zip) ?